tests/MongoDB.Driver.Core.Tests/Core/Connections/ConnectionInitializerTests.cs

```diff
@@ -39,6 +39,37 @@ public ConnectionInitializerTests()
             _subject = new ConnectionInitializer("test", new [] { new CompressorConfiguration(CompressorType.Zlib) });
         }
 
+        [Theory]
+        [ParameterAttributeData]
+        public void InitializeConnection_should_acquire_connectionId_from_isMaster_response([Values(false, true)] bool async)
+        {
+            var isMasterReply = MessageHelper.BuildReply(
+                RawBsonDocumentHelper.FromJson("{ ok: 1, connectionId: 1 }"));
```

Comment:
Consider the following formatting (with a space between each element):

    { ok : 1, connectionId : 1 }

Reply:
Done.